ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetBlockReceipts.java

```diff
@@ -95,8 +94,9 @@ private BlockReceiptsResult getBlockReceiptsResult(final Hash blockHash) {
             .map(
                 block ->
                     block.getTransactions().stream()
-                        .map(tx -> txReceipt(tx).get())
-                        .collect(Collectors.toList()))
+                            .map(this::txReceipt)
+                            .flatMap(Optional::stream)
+                            .collect(Collectors.toList()))
```

Comment:
can you add a test for the issue that this change is solving?

Reply:
Hi Macfarla, I actually don't know how to create tests. I am new to Java and currently learning. Can you please guide me?

Reply:
Have a look at EthGetBlockReceiptsTest - however on reflection I think it would need a fair amount of refactoring to add a test for this so I'm happy to merge as is.

Reply:
Hi @macfarla, 

I have run the `EthGetBlockReceiptsTest`, and it is passing. If you want me to run another test, please let me know the details.